src/utils/logger/logger-typescript.ts

```diff
@@ -108,16 +108,18 @@ export const loadTypeScriptDiagnostic = (tsDiagnostic: Diagnostic): d.Diagnostic
     header: 'TypeScript',
     code: tsDiagnostic.code.toString(),
     messageText: flattenDiagnosticMessageText(tsDiagnostic, tsDiagnostic.messageText),
-    relFilePath: null,
-    absFilePath: null,
+    columnNumber: undefined,
+    lineNumber: undefined,
+    relFilePath: undefined,
+    absFilePath: undefined,
```

Comment:
this and a few other changes are because of the slightly stricter typing of the `d.Diagnostic` interface